model/normalizer.go

```
@@ -226,3 +266,7 @@ func normMetricNameParse(name string) (string, bool) {
 
 	return string(res), true
 }
+
+func forceUtf8(s string) (string, error) {
```

I'd say it might be (insignificantly) more efficient to have only one decoder per function call. Maybe it would be best to inline this, but I don't feel strongly about it.

If you decide to keep it here, can we just call it `toUTF8` instead? I feel it would be much clearer. Force seems like a word that we would use with an operation that has the potential to fail but we are ignoring the error. I don't think this is the case.